### DISCUSSION TITLE
Is there something like a `PanelManager`?

### DISCUSSION THREAD

**Discussion by matthew-dean** [primefaces/community#1116] [PrimeVue]
When someone navigates to a new page, I'd like to close all open OverlayPanels. Some dialog components for Vue include something like a manager that will allow you to either access the group or emit actions for the group. Does something like that exist for PrimeVue overlays?

**Comment by tugcekucukoglu** [comment]
Is there a demo where we can replicate the issue? Were you able to find a case like this on showcase?
